src/common/components/NavMenu.js

```diff
@@ -38,25 +48,48 @@ export default class NavMenu extends Component {
     </NavLink>
   )
 
-  render() {
-    return (
-      <Drawer
-        open
-        containerStyle={{
-          top: 54,
-          paddingTop: 10,
-          width: this.isExpanded ? 256 : 64,
-          transition: 'width 200ms ease-in-out',
-          height: 'calc(100% - 54px)',
-          overflow: 'visible'
-      }}>
-        {this.renderLink('/', 'Home', 'home')}
-        {this.renderLink('/collection', 'Collection', 'view_module')}
-        {this.renderLink('/creator', 'Creator', 'add_circle_outline')}
-        {this.renderLink('/decks', 'Decks', 'view_list')}
-        {this.renderLink('/play', 'Play', 'videogame_asset')}
-        {this.renderLink('/about', 'About', 'info_outline')}
-      </Drawer>
-    );
-  }
+  renderExpandCollapseButton = () =>
+    <div
+      onClick={this.toggleExpanded}
+      style={{
+        textAlign: 'center',
+        cursor: 'pointer'
+      }}
+    >
+      <Tooltip
+        text={isFlagSet('sidebarCollapsed') ? 'Expand' : 'Collapse' }
+        place="right"
+      >
+        <FontIcon
+          className="material-icons"
+          style={{
+            fontSize: '0.6em',
+            color: '#666'
+          }}
+        >
+          {this.isExpanded ? <span>arrow_forward&zwnj;arrow_back</span> : <span>arrow_back&zwnj;arrow_forward</span>}
```

Comment:
[nit] I had to look up the purpose of `&zwnj;` maybe a utility method like

```javascript
const glueTogether = (...items) => items.join('&zwnj;');
glueTogether(arrow_forward, arrow_back);
```

Might be clearer?